lesson02/assignment/index.html

```diff
@@ -3,12 +3,36 @@
         <meta charset="UTF-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-        <link rel="stylesheet" href="css/style.css" />
+        <link rel="stylesheet" href="css/main.css" />
```

Comment:
Why did you create a main.scss instead of using the existing style.scss? If you want it to have a different name, just rename the file. No need to keep both, when only one is being used.